<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPLineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1:8086</string>
   </property>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>800</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="connectStateLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Not Connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="positionLineEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>160</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setValueLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="angleLineEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setValueButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SetValue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect State</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>倾角(°)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>位置(m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>小车质量(kg)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摆杆质量(kg)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.018</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>490</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摆杆转动惯量(kg*m/s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摆杆重心到关节的距离(m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>阻尼系数(N/m)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>控制器综合输出(V)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="controllerOutLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SlaveID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slaveIDLineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>photoLabel</zorder>
  <zorder>connectButton</zorder>
  <zorder>IPLineEdit</zorder>
  <zorder>label</zorder>
  <zorder>connectStateLineEdit</zorder>
  <zorder>positionLineEdit</zorder>
  <zorder>setValueLineEdit</zorder>
  <zorder>angleLineEdit</zorder>
  <zorder>setValueButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>label_8</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>controllerOutLineEdit</zorder>
  <zorder>label_11</zorder>
  <zorder>slaveIDLineEdit</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
